octobot_commons/constants.py

```diff
@@ -43,6 +43,17 @@
 CONFIG_FILE_EXT = ".json"
 CONFIG_FILE_SCHEMA = f"{CONFIG_FOLDER}/config_{SCHEMA}.json"
 
+# Config currencies
+CONFIG_CRYPTO_CURRENCIES = "crypto-currencies"
+CONFIG_CRYPTO_CURRENCY = "crypto-currency"
+CONFIG_CRYPTO_PAIRS = "pairs"
+CONFIG_CRYPTO_QUOTE = "quote"
+CONFIG_CRYPTO_ADD = "add"
+
+# Config notifications
+CONFIG_CATEGORY_NOTIFICATION = "notification"
```

Comment:
Why this is required to be in OctoBot-Commons ?

Reply:
(Also for subsequent comments)
Since these are used in octobot-notification and other modules (at least the web interface tentacle) I added them in common. However they are strings and enums, we can decide to put them in their sementic module but we need to be careful with circular dependancies

Reply:
I think it's relevant that OctoBot-Notifications and OctoBot-Interfaces import OctoBot-Services.

Reply:
Moreover, if this constants are put in OctoBot-Services, there shouldn't be a circular dependency because in my opinion OctoBot-Services should only be imported by OctoBot-Interfaces, OctoBot-Notifications.

Reply:
moved to OctoBot-Notifications